piplicenses.py

```diff
@@ -540,7 +540,8 @@ def create_output_string(args: CustomNamespace) -> str:
     sortby = get_sortby(args)
 
     if args.format_ == FormatArg.HTML:
-        return table.get_html_string(fields=output_fields, sortby=sortby)
+        html = table.get_html_string(fields=output_fields, sortby=sortby)
+        return html.encode('ascii', errors='xmlcharrefreplace').decode('ascii')
```

Comment:
Thanks for the PR. Looks good.

Linter was introduced in CI some time ago.

The following code styles are noted could you please correct the string? or `make lint` will also fix it.

```diff
+++ piplicenses.py	2022-12-08 11:34:58.939191 +0000
@@ -539,11 +539,11 @@
 
     sortby = get_sortby(args)
 
     if args.format_ == FormatArg.HTML:
         html = table.get_html_string(fields=output_fields, sortby=sortby)
-        return html.encode('ascii', errors='xmlcharrefreplace').decode('ascii')
+        return html.encode("ascii", errors="xmlcharrefreplace").decode("ascii")
```

Reply:
Done. Apologies, hadn't realised that was in place.